common-crafting.lic

```diff
@@ -409,6 +409,36 @@ module DRCC
     end
   end
 
+  def repair_tools(tools, bag, belt, town, settings, pickup = true)
+    tools = tools.to_a
+    repair_info = get_data('town')[town]['metal_repair']
+    DRCT.walk_to(repair_info['id'])
+    tools.each do |tool_name|
+      get_crafting_item(tool_name, bag, nil, belt, true)
+      case DRC.bput("give #{repair_info['name']}", /I don't repair those here/, /What is it/, /There isn't a scratch on that/, /Just give it to me again/, /I will not/, /I can't fix those/)
```

Comment:
How about using DRCI.give_item?, and expanding the messaging there to account for NPCs?

Reply:
yeah, that's a much better idea, i didn't think about the stow issue. I don't know what you mean by dealing with npcs thought.

Reply:
meant this comment on the next review item. This is doable, though..